src/DynamoCoreWpf/Views/Search/LibraryContainerView.xaml.cs

```diff
@@ -42,7 +42,6 @@ public SearchView(SearchViewModel searchViewModel, DynamoViewModel dynamoViewMod
             viewModel = searchViewModel;
             this.dynamoViewModel = dynamoViewModel;
 
-            DataContext = viewModel;
```

Comment:
Another problem:
if data context is not set, we have BindingExpressions in LibraryContainerView.
System.Windows.Data Error: 40 : BindingExpression path error: .......

Can we leave these binding errors not fixed?